exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = true; // finish this statement
```

Comment:
Hi Amanda, to use the declared variables instead of 'true and false' would be better here. you'll see that true and false will be displayed in terminal.
ex: 
  var moreStudentsThanMentors = studentCount > mentorCount;

Reply:
That is very true. Will do that next time, thank you Gulnihal.